src/frontend/screens/WebView/index.tsx

```diff
@@ -193,6 +204,7 @@ export default function WebView() {
         src={startUrl}
         allowpopups={trueAsStr}
         useragent="Mozilla/5.0 (Windows NT 10.0; WOW64; rv:70.0) Gecko/20100101 Firefox/70.0"
+        {...(preloadPath ? { preload: preloadPath } : {})}
```

Comment:
We should call setPreloads on a persistent session and then load the webview into that session in the future. It would create a better separation between frontend and backend,  also eliminate a lot of boilerplate, and be more easily extensible.

Reply:
we can refactor this later if we end up needing another webview with the same preload